docs/src/views/Theming.tsx

```diff
@@ -208,6 +208,106 @@ export default () => (
       <NavLink to="components/provider">Provider</NavLink> at the root of your app.
     </p>
 
+    <Header as="h2" content="Animations" />
+    <p>
+      You define animations in Stardust in a very similar way to CSS, by providing keyframes and
+      animation properties.
+    </p>
+
+    <ExampleSnippet
+      value={[
+        `<Provider`,
+        `  theme={{`,
+        `    animations: {`,
+        `      spinner: {`,
+        `        keyframe: {`,
+        `          from: {  transform: 'rotate(0deg)' },`,
+        `          to: { transform: 'rotate(360deg)' },`,
+        `        },`,
+        `        duration: '5s',`,
+        `        iterationCount: 'infinite'`,
+        `     },`,
+        `    },`,
+        `  }}`,
+        `>`,
+        ` ... `,
+        `</Provider>`,
+      ].join('\n')}
+    />
+
+    <p>
+      You can define the <code>animations</code> in a part of your render tree using the{' '}
+      <NavLink to="components/provider">Provider</NavLink>.
+    </p>
+    <p>
+      This is done with the Provider's <code>theme</code> prop. The animations are then applied
+      based on their name by using the <NavLink to="components/Animation">Animation</NavLink>{' '}
+      component, or the <code>animation</code> property available on all Stardust component. Here's
+      how we can use them in our components.
+    </p>
+    <ExampleSnippet
+      value={[
+        `<Provider`,
+        `  theme={{`,
+        `    animations: {`,
+        `      spinner: {`,
+        `        keyframe: {`,
+        `          from: {  transform: 'rotate(0deg)' },`,
+        `          to: { transform: 'rotate(360deg)' },`,
+        `        },`,
+        `        duration: '5s',`,
+        `        iterationCount: 'infinite'`,
+        `     },`,
+        `    },`,
+        `  }}`,
+        `>`,
+        `  <div>`,
+        `    <Animation name="spinner"><Icon name="user" circular /></Animation>`,
+        `    <Icon name="book" animation="spinner" circular/>`,
+        `  </div>`,
+        `</Provider>`,
+      ].join('\n')}
+      render={() => (
+        <div>
+          <Animation name="spinner">
+            <Icon name="user" circular />
+          </Animation>
+          <Icon name="book" animation="spinner" circular />
+        </div>
+      )}
+    />
+
+    <p>
+      You can also override some of the defined <code>animation</code> properties, by providing
```

Comment:
I would explicitly mention here that `keyframe` cannot be overriden and can be only defined in the `Provider`.

Reply:
Good point! Will be sure to mention it.